bigtable/src/main/scala/magnolify/bigtable/BigtableType.scala

```diff
@@ -16,26 +16,25 @@
 
 package magnolify.bigtable
 
-import java.nio.ByteBuffer
-import java.util.UUID
-import com.google.bigtable.v2.{Cell, Column, Family, Mutation, Row}
 import com.google.bigtable.v2.Mutation.SetCell
+import com.google.bigtable.v2.*
 import com.google.protobuf.ByteString
-import magnolia1._
-import magnolify.shared._
-import magnolify.shims._
+import magnolia1.*
+import magnolify.shared.*
+import magnolify.shims.*
 
+import java.nio.ByteBuffer
+import java.util.UUID
 import scala.annotation.implicitNotFound
-import scala.jdk.CollectionConverters._
-import scala.collection.compat._
+import scala.jdk.CollectionConverters.*
 
-sealed trait BigtableType[T] extends Converter[T, java.util.List[Column], Seq[SetCell.Builder]] {
+sealed trait BigtableType[T] extends Converter[T, Map[String, Cell], Seq[SetCell.Builder]] {
   def apply(v: Row, columnFamily: String): T =
     from(
       v.getFamiliesList.asScala
         .find(_.getName == columnFamily)
-        .map(_.getColumnsList)
-        .getOrElse(java.util.Collections.emptyList())
+        .map(_.getColumnsList.asScala.map(c => c.getQualifier.toStringUtf8 -> c.getCells(0)).toMap)
```

Comment:
we should probably keep the `Column` here instad of the cell in case we get extra colums with empty cells